s3fs/core.py

```diff
@@ -100,12 +99,15 @@ class S3FileSystem(AbstractFileSystem):
     client_kwargs : dict of parameters for the boto3 client
     requester_pays : bool (False)
         If RequesterPays buckets are supported.
-    default_block_size: None, int
+    default_block_size: int (None)
         If given, the default block size value used for ``open()``, if no
         specific value is given at all time. The built-in default is 5MB.
     default_fill_cache : Bool (True)
         Whether to use cache filling with open by default. Refer to
         ``S3File.open``.
+    default_cache_type : string ('bytes')
+        "bytes", "mmap" or "none". If given, the default cache_type value used
+        for ``open()``. The default is 'bytes'.
```

Comment:
(perhaps refer to the fsspec doc or module, since there could be more cache types later)

Reply:
updated wordings as you suggested